Structure_Engine/Modify/Flip.cs

```diff
@@ -69,6 +69,26 @@ public static Edge Flip(this Edge edge)
 
             return flipped;
         }
+
+        [Description("Flips normal of the Stem.")]
+        [Input("stem", "The Stem to flip.")]
+        [Output("stem", "The Stem with flipped Normal")]
+        public static Stem Flip(this Stem stem)
+        {
+            if (stem.IsNull())
+                return null;
+            if (stem.Normal.IsNull())
+            {
+                Base.Compute.RecordError("The Normal of the Stem is null and could not be flipped.");
+                return null;
+            }
```

Comment:
```suggestion
            if (stem.Normal.IsNull("The Normal of the Stem is null and could not be flipped.","Flip"))
                return null;
```
The `IsNull` has an overload where you can provide a message, so this can be simplified.

Reply:
Perfect , thanks!